src/components/ui/SearchableDropdown.svelte

```diff
@@ -36,11 +36,13 @@
   export let settingsIconTooltipPlacement: string = 'top';
 
   export function hideMenu() {
-    dispatch('hideMenu');
-    presetMenu.hide();
+    if (presetMenu) {
```

Comment:
there's a chance that the `presetMenu` doesn't exist?

Reply:
Haven't nailed down exactly why this happens yet but if you go to the plan merge review page and try to click on a diff the `presetMenu` for some reason does not yet exist.

Reply:
Gotcha! That makes sense. Do you mind also updating the type to reflect that it could be `undefined`, please?